src/nxdoc/web-ui/nuxeo-elements/nuxeo-elements-tutorial.md

```diff
@@ -106,36 +105,39 @@ Let's plug this application into the Nuxeo instance and change the hardcoded use
 
 ## Create a New Element
 
-1.  Create a new element named `doc-reader` to hold our logic, under the `src` directory. For simplicity, we can base
-    our element in one of those already provided by PSK:
+1.  Create a new element named `doc-reader` to hold our logic, under the `src` directory. For simplicity, we can create a new file called `my-doc-reader.js` in the folder `src` of the scaffolded structure provided by PSK.
```

Comment:
> When changing bower install to npm i, we're dropping the --save flag, which would store the dependency in bower.json/package.json. Are we sure about that?
AFAIK `npm i --save` is no longer needed since npm 5 (as you mentioned, it adds the package to package.json and to package-lock.json). So what we could do here was to do `--save-dev`. To keep it simple, in the context of these tutorials, I would say we could keep it as it is, but I can always change it if you think it is more explicit

Reply:
Yup, you're right on that one, my bad.

Reply:
> We're referencing maintenance-3.0.x in a bunch of places, even though this PR is going to be merged into the master doc branch. Shouldn't we use master instead? Wouldn't maintenance-3.0.x make more sense when backporting this (if that's the case) to the 2021 branch of the doc?

Regarding this one, I just confirmed with @manonlumeau and the best practice is to point out to a maintenance branch since the master one might be subject to several changes.